src/upload.ts

```diff
@@ -90,13 +90,15 @@ async function uploadVideo(videoJSON: Video) {
     const playlistName = videoJSON.playlist
     const videoLang = videoJSON.language
     const thumb = videoJSON.thumbnail
+    const keepDraw = videoJSON.keepDraw
```

Comment:
Why is the naming "draw" everywhere when youtube names it "draft" ?
"uploadAsDraft" maybe ?

Reply:
it's typo, corrected.